src/localized-strings.js

```diff
@@ -59,6 +59,8 @@ export const LocalizedStringId = {
   AMP_STORY_PAGINATION_BUTTON_PREVIOUS_PAGE_LABEL: '82',
   AMP_STORY_PAGE_ERROR_VIDEO: '65',
   AMP_STORY_PAGE_PLAY_VIDEO: '34',
+  AMP_STORY_PAUSE_BUTTON_LABEL: '85',
```

Comment:
Please update the `Next ID`

Also, shouldn't this one be `84`?

Reply:
84 is the "SCORE:" from the results component, but I also forgot to update that